src/quo2/components/list_items/saved_contact_address/style.cljs

```diff
@@ -0,0 +1,61 @@
+(ns quo2.components.list-items.saved-contact-address.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn- background-color
+  [{:keys [state customization-color]}]
+  (cond (or (= state :pressed) (= state :selected))
```

Comment:
should this not be themed? 🤔

Reply:
Hmm its just the customization color with different opacities with not difference on the themes, so I think not